ios/dtx_codec/connection.go

```diff
@@ -87,13 +96,28 @@ func NewGlobalDispatcher(requestChannelMessages chan Message, dtxConnection *Con
 	return dispatcher
 }
 
+// Dispatch to a MessageDispatcher of the Connection if set
+func (dtxConn *Connection) Dispatch(msg Message) {
+	msgDispatcher := dtxConn.MessageDispatcher
+	if msgDispatcher != nil {
+		log.Debugf("msg dispatcher found: %v", reflect.TypeOf(msgDispatcher))
```

Comment:
there is a format specifier that you can use so that you don't have to go through the `reflect` library
```suggestion
		log.Debugf("msg dispatcher found: %T", msgDispatcher)
```